src/main/resources/application.properties

```diff
@@ -12,4 +12,11 @@ spring.data.mongodb.host=localhost
 # App Config
 #server.port=8102
 spring.application.name=Chemistry
-#server.servlet.context-path=/user
\ No newline at end of file
+#server.servlet.context-path=/user
+
+pubchem.prolog=https://pubchem.ncbi.nlm.nih.gov/rest/pug
+pubchem.input=/compound/fastformula/
+pubchem.operation=/property/MolecularFormula,MolecularWeight,Title
+pubchem.output=/JSON
```

Comment:
These variables should be setup with the names the API uses: [from the tutorial](https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest-tutorial$_Toc458584410).

- I'll push a commit to this branch that has that update.